### ISSUE TITLE
lag and other distros

### ISSUE THREAD

**Issue by lollilol** [KrispyCamel4u/SysMonTask#8]
at first: thanks for this great program. i really appreciate it, i have not found another program that is as good as this. 
My problem that i have is that there is only a .deb.
I would really appreciate it if you would also provide a .rpm for Fedora/RHEL Users and an AUR for Arch users.

**Comment by lollilol** [comment]
also i have custom font settings that dont work because it uses the root user to run. One thing that i noticed lately is that (if you install vmware workstation) you can run "vmware-netcfg" and it asks in a Gnome GUI for admin permissions
![image](https://user-images.githubusercontent.com/38194372/109236279-9663f180-77cf-11eb-8f1c-d8cb14961539.png)
and then it also uses the custom fonts that i set. When i run it with sudo it doesnt. So that somehow works too.

Btw im not a coder, else i would probably do some PRs but im really not into coding lol

**Comment by KrispyCamel4u** [comment]
Hi,
Thanks, I will look into that custom font settings issue and I am currently looking for help for packaging for other distros.
Meanwhile, can you explain more about the "lag" that you are experiencing?

**Comment by lollilol** [comment]
sure, i figured out its only while using fakeroot. Everything is fine when just running it normally. But the issue was that it just kind of freezed all the time, it just wasnt reacting to my mouse clicks and also i think i couldn't move the window around. Im using Fedora 33 with Gnome on Wayland.

**Comment by RDKRACZ** [comment]
I would like to request a .appimage file. This way we could use SysMonTask on any distro before distribution-specific packages get released (if any). If you were thinking about snap or flatpak then please hold on because both are really bad for various reasons (flatpak is better IMO)

**Comment by dtantono** [comment]
Yes, I think deb or snap or flatpak package is good alternatives as well

**Comment by KrispyCamel4u** [comment]
Hi all,
The AUR and RPM package has been made and published. The instructions are in the readme.

**Comment by mkpankov** [comment]
It's cool seeing progress in this area.
However, supporting Fedora 35 which is still in development is not everything.
Rather than building for the most recent distributive, it's better to built for reasonably-old-enough distributive as package can then be installed on more recent distributives as well due to backward compatibility of libraries.
For example, I'm on Fedora 32 and package for Fedora 35 won't work on my system. Contrary, package for Fedora 32 will work on 35.

**Comment by KrispyCamel4u** [comment]
> It's cool seeing progress in this area.
> However, supporting Fedora 35 which is still in development is not everything.
> Rather than building for the most recent distributive, it's better to build for reasonably-old-enough distributive as package can then be installed on more recent distributives as well due to backward compatibility of libraries.
> For example, I'm on Fedora 32 and package for Fedora 35 won't work on my system. Contrary, package for Fedora 32 will work on 35.

The package build request has already been sent. It will take some time to appear in  f33 and f34 repos.

**Comment by lollilol** [comment]
If you were thinking about snap or flatpak then please hold on because both are really bad for various reasons (flatpak is better IMO)
AppImages are cool because it runs everywhere without you having to install anything, but flatpaks are also really cool, they just work and for example in the spotify package, they fixed a bug thats still to this day in the repo/binary version. Im using flatpak on fedora, maybe its just fedora that integrates it really well, since its also preinstalled i think, but flatpak isnt really that bad as you think.

**Comment by RDKRACZ** [comment]
I use flatpacks too but don’t like them mainly because themes don’t work and also sometimes some features. Snaps are just another Cannonical garbage. Appimages are great but sadly they don’t integrate with system well eg you can’t just type it’s name in the xfce app menu to launch it.

**Comment by mkpankov** [comment]
So, Fedora has very old 1.1.1 which crashes a lot
```
➜  ~ rpm -qa sysmontask
sysmontask-1.1.1-0.5.beta.b.fc33.noarch
```
Is there a way to ensure automatic builds of recent versions? I'd like 1.3.9 but I don't want to install from source.

**Comment by KrispyCamel4u** [comment]
> So, Fedora has very old 1.1.1 which crashes a lot
> 
> ```
> ➜  ~ rpm -qa sysmontask
> sysmontask-1.1.1-0.5.beta.b.fc33.noarch
> ```
> 
> Is there a way to ensure automatic builds of recent versions? I'd like 1.3.9 but I don't want to install from source.

Hi,
You can track the status of rpm build for sysmontask [here](https://bodhi.fedoraproject.org/updates/?packages=sysmontask)
Also, I have included the RPM package file in the [*releases*](https://github.com/KrispyCamel4u/SysMonTask/releases/tag/v1.3.9).

**Comment by Danny3** [comment]
> I use flatpacks too but don’t like them mainly because themes don’t work and also sometimes some features. Snaps are just another Cannonical garbage. Appimages are great but sadly they don’t integrate with system well eg you can’t just type it’s name in the xfce app menu to launch it.

AppImages can be integrated and searched for in the start menu, if you can install AppImageLauncher.

**Comment by iMonZ** [comment]
> > I use flatpacks too but don’t like them mainly because themes don’t work and also sometimes some features. Snaps are just another Cannonical garbage. Appimages are great but sadly they don’t integrate with system well eg you can’t just type it’s name in the xfce app menu to launch it.
> 
> AppImages can be integrated and searched for in the start menu, if you can install AppImageLauncher.

Maybe snapcraft is a better idea?
